frontend/app/templates/components/badge-table.hbs

```diff
@@ -1,19 +1,19 @@
-<table class="ui table">
+<table class="ui sortable table">
     <thead>
         <tr>
             <th>Badge Creator Name</th>
             <th>Badge Name</th>
             <th>Created At</th>
-            <th>Deleted At</th>
-            <th class="center aligned">Actions</th>
+            <th class="no-sort">Deleted At</th>
```

Comment:
Why not sort also from `Deleted At` ?

Reply:
@yashLadha  Applied Sort in `Deleted at` also. :+1: